src/Node/Stream.purs

```diff
@@ -65,6 +66,9 @@ type Writable r = Stream (write :: Write | r)
 -- | A duplex (readable _and_ writable stream)
 type Duplex = Stream (read :: Read, write :: Write)
 
+-- | A passThrough (readable _and_ writable stream)
+type PassThrough = Stream (read :: Read, write :: Write)
```

Comment:
Different in that `PassThrough` doesn't perform any transform on the data. 

However, I saw how it can be defined in the tests, so I'll close this.